test/box/select.test.lua

```diff
@@ -148,3 +148,15 @@ collectgarbage('collect')
 collectgarbage('restart')
 lots_of_links
 s:drop()
+
+-- gh-6501 attempt to index local 'opts' (a number value)
```

Comment:
Please add a changelog entry. For example, see https://github.com/tarantool/tarantool/commit/2bbd13b45e500912cf2412e7ea618d126ecaf4f9 or any other commit that adds a file to `changelog/unreleased` directory.

Reply:
Fixed it too.